utils/histogram_metrics_helper.hh

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2020 ScyllaDB
+ *
+ * Modified by ScyllaDB
+ */
+
+/*
+ * This file is part of Scylla.
+ *
+ * Scylla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Scylla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <seastar/core/metrics_types.hh>
+#include "seastarx.hh"
+#include "estimated_histogram.hh"
+
+
+template<uint64_t Min, uint64_t Max, std::vector<uint64_t>::size_type NumBuckets>
+seastar::metrics::histogram to_metrics_histogram(const utils::pseudo_floatingpoint_histogram<Min, Max, NumBuckets>& hist) {
+    seastar::metrics::histogram res;
```

Comment:
This can also work by passing the template parameters as non-template parameters to an out-of-line funciton.

Reply:
What about this?

Reply:
You can still see the context if you go to the individual commit.